sphinx/ext/autodoc/__init__.py

```diff
@@ -1538,15 +1538,3 @@ def setup(app):
     app.add_event('autodoc-skip-member')
 
     return {'version': sphinx.__display_version__, 'parallel_read_safe': True}
-
-
-class testcls:
```

Comment:
Wow..  It seems that it has never been referenced historically https://github.com/sphinx-doc/sphinx/commit/ef674a20a6af061182720594b10d5c92c9e10087